lib/sqlalchemy/exc.py

```diff
@@ -19,16 +19,16 @@
 _version_token = None
 
 
-class SQLAlchemyError(Exception):
-    """Generic error class."""
+class _CodeStrMixin(object):
```

Comment:
I would call this HasDescriptionCode, and we likely don't need to extend "object" anymore, as it's a mixin so Py2 isn't going to care.